### PR TITLE
[SPARK-30342][SQL][DOC]Update LIST FILE/JAR command Documentation

### PR DESCRIPTION
**Pull request by iRakson**
### What changes were proposed in this pull request?
Updated the document for LIST FILE/JAR command.

### Why are the changes needed?
LIST FILE/JAR can take multiple filenames as argument and it returns the files which were added as resources.

### Does this PR introduce any user-facing change?
Yes. Documentation updated for LIST FILE/JAR command

### How was this patch tested?
Manually

### CODE REVIEW

**Comment by iRakson** [comment]
cc @maropu @cloud-fan

**Comment by cloud-fan** [comment]
ok to test

**Comment by maropu** [comment]
In addition, can you add more examples for quoted paths of ADD FILE/JAR? https://github.com/apache/spark/pull/26779

**Comment by iRakson** [comment]
Can you review ADD directory #26863. If thats ok then i can add that here as well.

**Comment by maropu** [comment]
I think you don't need to block this pr by #26863. If necessary to update the doc for the fix in #26863, you can include the fix in the PR.

**Review by maropu — APPROVED** [review]
LGTM cc: @HyukjinKwon

File: docs/sql-ref-syntax-aux-resource-mgmt-add-file.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ ADD FILE file_name
 ### Examples
 {% highlight sql %}
 ADD FILE /tmp/test;
+ADD FILE "/path/to/file/abc.txt";
+ADD FILE '/another/test.txt';
+ADD FILE "/path with space/abc.txt";
```
Comment (srowen) [inline docs/sql-ref-syntax-aux-resource-mgmt-add-file.md]: Does this work yet?

Comment (iRakson) [reply docs/sql-ref-syntax-aux-resource-mgmt-add-file.md]: Yeahh. Its fixed in #26773

**Comment by srowen** [comment]
Merged to master

**Comment by iRakson** [comment]
Thanks @srowen @maropu

**Comment by HyukjinKwon** [comment]
LGTM too. Thanks for pinging me.

**Pull request closed without merging**